app/templates/components/events/view/overview/speaker-session.hbs

```diff
@@ -0,0 +1,116 @@
+<div class="content">
+  <div class="header">{{t 'Sessions And Speakers'}}</div>
```

Comment:
Sessions ans Speakers

Reply:
Only do this change, everything else is working fine.